core/src/main/java/hudson/util/TimeUnit2.java

```diff
@@ -65,13 +65,12 @@
  * implementation will be able to notice the passage of time at the
  * same granularity as the given <tt>TimeUnit</tt>.
  *
- * @since 1.5
```

Comment:
This and the others are not referring to Jenkins versions, so are unhelpful.

I hope deleting comments is a legal operation in copied code.